example/example.go

```
@@ -45,6 +45,22 @@
 	// PodLink()
 }
 
+func ResourceCount() {
+	sm, err := kom.DefaultCluster().Status().GetResourceCountSummary(10)
+	if err != nil {
+		fmt.Println(err)
+	}
+	for gvr, count := range sm {
+		klog.V(6).Infof("%-50s   %-10d\n", gvr.String(), count)
+	}
+	sm, _ = kom.DefaultCluster().Status().GetResourceCountSummary(10)
```

golangci-lint (1.64.8)

ineffectual assignment to sm

(ineffassign)